app/controllers/manage_cmr_controller.rb

```diff
@@ -7,6 +7,13 @@ class ManageCmrController < PagesController
   before_filter :check_if_system_acl_administrator, only: :show
   before_filter :check_if_current_provider_acl_administrator, only: :show
 
+  # These are json respones for ajax calls that user wouldnt get to without being logged in.
+  skip_before_filter :is_logged_in, only: [
```

Comment:
should you add this for the `get_all_collections` endpoint in PermissionsController also?